src/common.ts

```diff
@@ -1,13 +1,37 @@
 import { KeyStore } from 'keystore-idb/types'
+import ipfs from './ipfs'
 
 
-// CONSTANTS
+// 🏔
 
 
 export const API_ENDPOINT = 'https://runfission.com'
 
 
 
+// API
+
+
+const apididcache = {
+  didKey: "",
+  lastFetched: 0,
+}
+
+
+export async function apiDidKey(): Promise<string> {
```

Comment:
I think we have different conceptions of what goes in `common`. 

I've mostly been using it for "vanilla javascript helpers", typechecks, string encoding, etc. You can check that out [here](https://github.com/fission-suite/ts-sdk/pull/34/files)

I'm a bit worried about this becoming too much of a "grab bag" file with a bit of everything in it.

I'd say we either:
- split up `common` into submodules (ie `common/base64`, `common/type-checks`, `common/dns`)
- ditch the `common` name all together. `util` might be a better name for what I've been using `common` for, and dns-lookups & caching and what not could exist in its own file

Thoughts? I don't have a strong opinion, but figure we should organize this before ti gets to hodge-podgey

Reply:
Yeah, with all that combined, it does get to be a lot. I named it `common` because these are functions that are used across multiple modules (ie. modules have, or could have, these things in common). I would use `util` if it's something more specific to a specific module. Does that make sense? 🤔 

I'll split it up into submodules 👍